feat: move docs to mkdocs (#824)
Mozilla-Ocho/llamafile: docs/example_llamafiles.md

## Other example llamafiles

> We also provide example llamafiles for other models, so you can easily

Reviewer: ```suggestion
We provide example llamafiles for a variety of models, so you can easily
```
